javaparser-testing/src/test/java/com/github/javaparser/printer/PrettyPrinterTest.java

```diff
@@ -97,21 +99,50 @@ public void printUseTestVisitor(){
     
     @Test
     public void prettyColumnAlignParameters() {
+        PrettyPrinterConfiguration config = new PrettyPrinterConfiguration();
+        config.setIndent("\t");
+        config.setColumnAlignParameters(true);
```

Comment:
should we have two versions of these test? With and without the column align parameters flag set

Reply:
@tarilabs could you address this question? It's policy not to merge PR's with open questions.

Reply:
@ftomassetti trying to address this question but a bit undecided: the full test suite (I mean all the other tests but this method) are actually testing "without" this flag. Proof is, if I force this flag as the default, a few tests break in string comparison. Therefore, the full test suite is acting as "flag=false", while this test is ensuring the pretty printer is acting as required when the "flag=true".

Anyway, if we prefer to have a specific test for the highlighted case in both variant, kindly let me know if you prefer for it to be:

- testing the other variant within the existing test method
- as a separate test method

(I would refrain from using JUnit "parametrized" because it would require the combinatorial of the several options..)

I have no problem adding additional test and assertion, just kindly let me know which strategy is preferred ;-) thanks!

Reply:
As a separate test method would be perfect. Thank you for your work on this one!